spec/vcr_cassettes/manageiq/providers/azure_stack/cloud_manager/refresher_targeted/V2017_03_09-resource_group.yml

```diff
@@ -0,0 +1,776 @@
+- request:
+    method: get
+    uri: https://AZURE_STACK_HOST/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/resources?api-version=2016-02-01
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - Ruby/2.5.6 (x86_64-linux) ms_rest/0.7.4 ms_rest_azure/0.11.0 Azure-SDK-For-Ruby
+        azure_mgmt_resources/0.17.2 Profiles/V2017_03_09/Resources/Mgmt
+      Content-Type:
+      - application/json; charset=utf-8
+      Accept:
+      - application/json
+      Accept-Language:
+      - en-US
+      X-Ms-Client-Request-Id:
+      - 126a268d-e1d4-4ab1-9953-8522e8105eb7
+      Authorization:
+      - Bearer AZURE_STACK_TOKEN
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Cache-Control:
+      - no-cache
+      Pragma:
+      - no-cache
+      Content-Type:
+      - application/json; charset=utf-8
+      Expires:
+      - "-1"
+      X-Ms-Ratelimit-Remaining-Subscription-Reads:
+      - '14998'
+      X-Ms-Request-Id:
+      - 1344f7ce-b739-4393-85e1-be4b24119451
+      X-Ms-Correlation-Request-Id:
+      - 1344f7ce-b739-4393-85e1-be4b24119451
+      X-Ms-Routing-Request-Id:
+      - WESTUS:20191003T064000Z:1344f7ce-b739-4393-85e1-be4b24119451
+      Strict-Transport-Security:
+      - max-age=31536000; includeSubDomains
+      X-Content-Type-Options:
+      - nosniff
+      Date:
+      - Thu, 03 Oct 2019 06:40:00 GMT
+      Content-Length:
+      - '1219'
+    body:
+      encoding: UTF-8
+      string: '{"value":[{"id":"/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/providers/Microsoft.Compute/virtualMachines/demoVm","name":"demoVm","type":"Microsoft.Compute/virtualMachines","location":"westus"},{"id":"/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/providers/Microsoft.Network/networkInterfaces/demoNic0","name":"demoNic0","type":"Microsoft.Network/networkInterfaces","location":"westus"},{"id":"/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/providers/Microsoft.Network/networkSecurityGroups/demoSecurityGroup","name":"demoSecurityGroup","type":"Microsoft.Network/networkSecurityGroups","location":"westus"},{"id":"/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/providers/Microsoft.Network/virtualNetworks/demoNetwork","name":"demoNetwork","type":"Microsoft.Network/virtualNetworks","location":"westus"},{"id":"/subscriptions/AZURE_STACK_SUBSCRIPTION/resourceGroups/demo-resource-group/providers/Microsoft.Storage/storageaccounts/demostorageaccount","name":"demostorageaccount","type":"Microsoft.Storage/storageaccounts","location":"westus"}]}'
+    http_version: 
+  recorded_at: Thu, 03 Oct 2019 06:40:00 GMT
+- request:
+    method: get
+    uri: https://AZURE_STACK_HOST/subscriptions/AZURE_STACK_SUBSCRIPTION/resourcegroups/demo-resource-group?api-version=2016-02-01
```

Comment:
This request is in the cassette twice, but all tests are still green if I just remove it. I'm wondering how this happened, is it perhaps a leftover of some debugging? If yes, then just delete and re-record the cassette and it should be fixed